services/download/src/main/java/app/coronawarn/server/services/download/FederationBatchProcessor.java

```diff
@@ -160,6 +160,9 @@ private boolean isEfgsEnforceDateBasedDownloadAndNotSeen(String batchTag) {
     logger.info("Processing batch for date {} and batchTag {}.", date, batchTag);
     try {
       BatchDownloadResponse response = federationGatewayDownloadService.downloadBatch(batchTag, date);
+      if (config.isBatchAuditEnabled()) {
+        federationGatewayDownloadService.auditBatch(batchTag, date);
```

Comment:
Noticed that there are two separate catch-clauses for this block for `BatchDownloadException` and `Exception`. They seem to be identical, maybe it's worth to merge them together. It may be worth to also mention the `BatchAuditException` in a catch-clause in order to make clear what might go wrong?